ivy_tests/test_ivy/test_frontends/test_numpy/test_ndarray/test_ndarray.py

```diff
@@ -3622,3 +3622,44 @@ def test_numpy___invert__(
         method_flags=method_flags,
         on_device=on_device,
     )
+    
+# var
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="numpy.array",
+    method_name="var",
+    dtype_x_axis=helpers.dtype_values_axis(
+        available_dtypes=helpers.get_dtypes("float"),
+        valid_axis=True,
+    ),
+)
+def test_numpy_ndarray_var(
+    dtype_x_axis,
+    frontend_method_data,
+    init_flags,
+    method_flags,
+    frontend,
+    backend_fw,
+    on_device,
+):
+    input_dtypes, x, axis = dtype_x_axis
+    helpers.test_frontend_method(
+        init_input_dtypes=input_dtypes,
+        method_input_dtypes=input_dtypes,
+        init_all_as_kwargs_np={
+            "object": x[0],
+        },
+        method_all_as_kwargs_np={
+            "axis": axis,
+            "dtype": "float64",
+            "out": None,
```

Comment:
The testing pipeline will take care of testing the `out` argument, no need to pass that as `None` here.

Reply:
method_all_as_kwargs_np={
            "axis": axis,
},

Is it fine?

Reply:
Just remove `"out":None`